articles/aks/open-service-mesh-troubleshoot.md

```diff
@@ -378,51 +325,23 @@ We must have the following installed on the cluster:
 Get the versions of the CRDs installed with this command:
 
 ```azurecli-interactive
-for x in $(kubectl get crds --no-headers | awk '{print $1}' | grep 'smi-spec.io'); do
-    kubectl get crd $x -o json | jq -r '(.metadata.name, "----" , .spec.versions[].name, "\n")'
-done
+osm mesh list
```

Comment:
Lets change the ## Verify SMI CRDs to ## Verify OSM CRDs: and add a section for SMI crds and the other crds. Please update the list based on the crds expected with OSM v0.11.1